build.gradle

```diff
@@ -91,8 +91,10 @@ javadoc {
     options.addStringOption("doctitle").value = ""
     options.addStringOption("header").value = ""
 
-    options.links("http://docs.oracle.com/javase/7/docs/api/")
-    options.links("http://www.reactive-streams.org/reactive-streams-${reactiveStreamsVersion}-javadoc/")
+    options.links(
+        "https://docs.oracle.com/javase/7/docs/api/",
```

Comment:
Any reason not to use 8 or 9?

Reply:
No particular reason.

Reply:
Since RxJava 2.x is compatible with JDK 6, should we link 6?

Reply:
Well, the Javadoc is built with Java 8 and the JDK 6 docs look ugly to me anyway. Style-wise, 8 would be more seamless, 9 would be nicer due to the search box. Anything 7+ may trigger confusion about the JDK's default methods not available in Android or the misunderstanding that it's only the docs that link to Java 9, not that RxJava now depends on Java 9.